src/lib/components/primitives/input/input.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+    import type { HTMLInputAttributes } from "svelte/elements";
+    import type { WithElementRef } from "bits-ui";
+    import { cn } from "$lib/utils.js";
+
+    let {
+        ref = $bindable(null),
+        value = $bindable(),
+        class: className,
+        ...restProps
+    }: WithElementRef<HTMLInputAttributes> = $props();
+</script>
+
+<input
+    bind:this={ref}
+    class={cn(
+        "border-input bg-background ring-offset-background placeholder:text-muted-foreground flex h-10 w-full rounded-md border px-3 py-2 text-default file:border-0 file:bg-transparent file:text-sm file:font-medium focus-visible:outline-none disabled:cursor-not-allowed disabled:opacity-50",
```

Comment:
Let's use the theme that we already defined.
```suggestion
        "border-input-border-slate bg-background ring-offset-background placeholder:text-placeholder flex h-fit w-full rounded-md border px-4 py-2 text-default file:border-0 file:bg-transparent file:text-sm file:font-medium focus-visible:outline-none disabled:cursor-not-allowed disabled:opacity-50",
```

I also noticed that the `text-placeholder style doesn't use the predefined color for it, can you change that:
```pcss
.text-placeholder {
    @apply text-base font-medium tracking-normal text-input-placeholder-slate italic;
}
```

Reply:
I adapted the .text-placeholder style as well as the input styling (matching the theme from Figma).